fastify.js

```diff
@@ -7,9 +7,7 @@ const http = require('http')
 const https = require('https')
 const Middie = require('middie')
 const runHooks = require('fastseries')()
-
-var shot = null
-try { shot = require('shot') } catch (e) { }
+const Inject = require('light-my-request')
```

Comment:
Why is it capitalized ?

Reply:
Just syntactic sugar, I didn't like `lightMyRequest`, so I called it `Inject`.
It is capitalized because we already have a function called `inject`, which is the api itself.

Reply:
Commonly the capitalized words are used for classes or constructor functions.
I don't like the capitalized words if we don't call `new` on it